### PR TITLE
[ch28213] Swagger deprecated gives error on permissions

### PR DESCRIPTION
**Pull request by emaciupe**

### CODE REVIEW

File: django_api/etools_prp/apps/indicator/views.py [hunk 1]
```diff
@@ -310,6 +310,8 @@ class ReportableLocationGoalBaselineInNeedAPIView(ListAPIView, UpdateAPIView):
     def check_permissions(self, request):
         super().check_permissions(request)
         reportable_id = self.kwargs.get('reportable_id')
+        if not reportable_id and request.path == '/api/docs/':
```
Comment (emaciupe) [inline django_api/etools_prp/apps/indicator/views.py:313]: @domdinicola this is for https://dev.partnerreportingportal.org/api/docs/ error. Is there a better way to know the request is from swagger? for yasg there is the 'swagger_fake_view' which is handy. i couldn't find something similar for the rest swagger we're using.

Comment (domdinicola) [reply django_api/etools_prp/apps/indicator/views.py:313]: ... I would just change this:

```
        if request.path == '/api/docs/':
            return
        reportable_id = self.kwargs.get('reportable_id')
        .....

```

Comment (emaciupe) [reply django_api/etools_prp/apps/indicator/views.py]: done

**Comment by domdinicola** [comment]
@emaciupe extract swagger url as settings

**Review by domdinicola — APPROVED** [review]

**Pull request merged**